db/migrate/20230127173249_truncate_user_status_to_100_characters.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class TruncateUserStatusTo100Characters < ActiveRecord::Migration[7.0]
+  def up
+    execute "UPDATE user_statuses SET description = left(description, 100)"
+    execute "UPDATE user_statuses SET emoji = left(emoji, 100)"
```

Comment:
Do we even support emojis with more than one "character"? 🤔 

Should this column be a `character(1)` instead?

Reply:
We store emoji name as a string here, for example:

"paw_prints"
"palm_tree"
"desert_island"

I am not what's the longest possible name of emoji.

Reply:
Right, nevermind. I thought we were storing the unicode value.